generators/docs/generator.go

```diff
@@ -220,8 +220,13 @@ func (s *schemaType) IsNested() bool {
 	return len(s.Properties) > 0 || (s.Items != nil && s.Items.IsNested())
 }
 
-// reInlineCode finds all code-like text
-var reInlineCode = regexp.MustCompile(`'(\S+)'`)
+var (
+	// reInlineCode to wrap inline code into backticks
+	reInlineCode = regexp.MustCompile(`'(\S+)'`)
+
+	// reAdmonitions to render markdown admonitions
+	reAdmonitions = regexp.MustCompile(`(?i)((?:note|tip|info|warning|danger|bug|success|example) "[^"]*"):\s*(.+)`)
```

Comment:
is it possible to apply some markdown library instead, or will it be too much?

Reply:
It's not markdown. It is a plain text that I try to turn into markdown.